src/platform/utilities/tests/sso.unit.spec.js

```diff
@@ -82,11 +107,23 @@ describe('checkAutoSession', () => {
       'v1',
       null,
       null,
-      { authn: 'mhv', inbound: 'true' },
+      { authn: 'myhealthevet', inbound: 'true' },
       'sso-automatic-login',
     );
   });
 
+  it('should not auto login if user is logged out, they have a PIV SSOe session and dont need to force auth', async () => {
+    const sandbox = sinon.createSandbox();
+    sandbox.stub(profUtils, 'hasSession').returns(false);
+    sandbox.stub(profUtils, 'hasSessionSSO').returns(true);
+    sandbox.stub(forceAuth, 'getForceAuth').returns(undefined);
+    setKeepAliveResponse(global.fetch.onFirstCall(), 900, '33');
```

Comment:
Dumb question: Is `'33'` a valid `csid` value, or is `'33'` just a random key, like `'foo'`, that will return `undefined` on the following object?
```
{
  DSLogon: 'dslogon',
  mhv: 'myhealthevet',
  idme: resp.headers.get('va_eauth_authncontextclassref'),
}[resp.headers.get('va_eauth_csid')]
```

Reply:
33 is a PIV authenticated user, which would yield an empty authncontext. So while it's technically nonrandom, we want to ignore it in the same way as a random

Reply:
Cool. Thank you for that explanation.